### ISSUE TITLE
[Bug]: No error is reported when the sys tenant grants the grant account permission to the role

### ISSUE THREAD

**Issue by heni02** [matrixorigin/matrixone#5636]
### Is there an existing issue for the same bug?

- [X] I have checked the existing issues.

### Environment

```Markdown
- Version or commit-id (e.g. v0.1.0 or 8b23a93):
- Hardware parameters:
- OS type:
- Others:
```

### Actual Behavior

![image](https://user-images.githubusercontent.com/113406637/195832383-ab52b8f1-8403-49d2-9385-dee921a8dddf.png)

**Comment by daviszhen** [comment]
it was fixed at #5691

**Comment by heni02** [comment]
confirm,closed
![image](https://user-images.githubusercontent.com/113406637/196845337-07599d7c-a310-4285-9cb6-98df61cb8554.png)
